addons/purchase/models/purchase.py

```diff
@@ -558,9 +548,10 @@ def _inverse_qty_received(self):
 
     @api.model
     def create(self, values):
-        if values.get('display_type', self.default_get(['display_type'])['display_type']):
-            values.update(product_id=False, price_unit=0, product_uom_qty=0, product_uom=False, date_planned=False)
-
+        order_id = values.get('order_id')
+        order = self.env['purchase.order'].browse(order_id)
+        if order.date_planned:
+            values['date_planned'] = order.date_planned
```

Comment:
why not after the calls to super? you woudln't need ot browse

Reply:
>  you woudln't need ot browse

Doing this will call write inside create

Reply:
@sle done.